Sources/Session.swift

```diff
@@ -12,7 +12,17 @@ public struct WCURL: Hashable {
     public var bridgeURL: URL
     public var key: String
 
-    public init(topic: String, version: String, bridgeURL: URL, key: String) {
+    public static let defaultBridgeURL = URL(string: "https://safe-walletconnect.gnosis.io")!
+
+    public var absoluteString: String {
+        let bridge = bridgeURL.absoluteString.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed) ?? ""
+        return "wc:\(topic)@\(version)?bridge=\(bridge)&key=\(key)"
```

Comment:
The `topic`, `version, and`key` must also be percent-encoded.

Reply:
At this point, it is not needed.